helmfile/values/harbor.yaml.gotmpl

```diff
@@ -66,9 +66,9 @@ persistence:
       authurl: {{ $swift.authUrl }}
       region: {{ $swift.region }}
       container: {{ .Values.objectStorage.buckets.harbor }}
-      {{- if hasKey $swift "domainId" }}
+      {{- if $swift.domainId }}
       domainid: {{ $swift.domainId }}
-      {{- else if hasKey $swift "domainName" }}
+      {{- else if  $swift.domainName }}
```

Comment:
Isn't there a reason for having `hasKey`? Like if it's not set there's no error (or something like that)

Reply:
maybe @aarnq can answer that
but with `haskey` it will always return true as the keys exists in the default configs with `set-me`

Reply:
Hmm. The reason we can't have `hasKey` is because you should only specify either `domainId` or `domainName` and the init scripts will (for Cleura) add both into the default config so the key will always exist regardless of what you set it to.

I guess a way to have both is to use `if getOrNil $swift "domainId"`.

Reply:
should i so some tests with `getOrNil` instead?

Reply:
Yes if you have the time for it, else I can create issue to look at these things in general.

Reply:
done